components/04-page-layouts/_base.twig

```diff
@@ -0,0 +1,32 @@
+{% set layout__base_class = 'layout-container' %}
+{% set content__base_class = 'main' %}
+{% set sidebar__base_class = 'sidebar' %}
+{% set main__base_class = 'main-content' %}
+
+<div {{ bem(layout__base_class) }}>
+  {# Page Header #}
+  {% block page__header %}
+    {% include "@page-layouts/placeholder/placeholder.twig" with {
+      placeholder: 'Header',
+    } %}
+  {% endblock %}
+  {# System #}
+  {% block page_system %}{% endblock %}
+  <div {{ bem(content__base_class) }}>
+    <a id="main-content" tabindex="-1"></a>{# link is in html.html.twig #}
```

Comment:
Needs a skip to main content in the header to jump here.

Reply:
@ModulesUnraveled second this feedback. I think you can migrate the id to the `<main>` instead of using a hidden `<a>` here. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main

Reply:
The skip link will actually be in a totally different file (a drupal templates called `html.html.twig`).

The code in this PR will be implemented in the `page.html.twig` Drupal template which will be inserted as the `{{ page }}` element in this example `html.html.twig`:
```
<!DOCTYPE html>
<html>
  <head>
    <title>{{ head_title|safe_join(' | ') }}</title>
  </head>
  <body>
    {# Keyboard navigation/accessibility link to main content section in page.html.twig. #}
    <a href="#main-content" class="visually-hidden focusable skip-link">
      {{ 'Skip to main content'|t }}
    </a>
    {{ page }}
  </body>
</html>
```

(I've simplified it for brevity, but things like google analytics tracking code, etc. will also e here.)

Not sure if that makes sense or not, but the point is the actual skip link will be in another file. This file just needs to have the target where the skip link takes the user to.

I do agree that the id doesn't need to be on an `<a>` tag, though, so I'll move the ID up to the content element.

Reply:
I see, that makes sense! It's interesting to see how everything comes together.